ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -10,18 +10,19 @@ const KEY_LENGTH = 32
 export interface AccountValue {
   id: string
   name: string
-  spendingKey: string
+  spendingKey?: string
   incomingViewKey: string
   outgoingViewKey: string
   publicAddress: string
 }
 
 export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
```

Comment:
This creates a null placeholder value for the spending key in a serialized account. It was either this or have a variable size serialized account. I tried out both options and this looked cleaner/easier to reason about IMO.

Reply:
This seems OK to me. We use bitflags in a lot of serialized values to handle optional fields. We could do the same here for consistency, but that would require data migration ⚖️

Reply:
Yea I think I should follow the standard pattern of bit flags since that seems like the current pattern. I propose we ship it this way, then I can make migration/update. Otherwise this PR is gonna get even bigger. Wdyt?

Reply:
That sounds good to me 👍